### ISSUE TITLE
phase 7 accessibility issues and missing components.

### ISSUE THREAD

**Issue by WRH-05** [WRH-05/CelecNetworkR#6]
the phase 7 page is missing the nav menu button from the other pages to navigate to the other 7 pages, and it's also missing a title similair to phase4 page.
